packages/grid/x-data-grid-pro/src/components/DataGridProVirtualScroller.tsx

```diff
@@ -425,6 +427,9 @@ const DataGridProVirtualScroller = React.forwardRef<
         <GridVirtualScrollerRenderZone {...getRenderZoneProps()}>
           {mainRows}
         </GridVirtualScrollerRenderZone>
+        <GridVirtualScrollerRenderZone {...getFocusRenderZoneProps()}>
+          {getFocusedRow()}
```

Comment:
I've noticed scroll jump when checkbox cell is focused.
Doesn't happen if regular cell is focused though.


https://user-images.githubusercontent.com/13808724/192770394-33c62cb7-ba94-4bf3-a604-3c99b7eeede0.mov